docs/src/pages/getting-started/page-layout-examples/pricing/Pricing.js

```diff
@@ -28,9 +28,9 @@ const styles = theme => ({
   },
   layout: {
     width: 'auto',
-    marginLeft: theme.spacing.unit * 3,
-    marginRight: theme.spacing.unit * 3,
-    [theme.breakpoints.up(900 + theme.spacing.unit * 3 * 2)]: {
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(3),
+    [theme.breakpoints.up(900 + theme.spacing(3) * 2)]: {
```

Comment:
Is this the same as `900 + theme.spacing(6)` ?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

Reply:
We need to create a core component to abstract this logic :).